### DISCUSSION TITLE
Trigger update from Github Actions instead of interval

### DISCUSSION THREAD

**Discussion by nemanjam** [containrrr/watchtower#1185] [Q&A] [answered]
I have Github Actions workflow to build and push Docker production image for an app to Docker hub. I am the only one that is working on it and there will be lets say one release per week. Normally I want production server updated as soon as new image is built and pushed successfully.

But I can only set small interval argument and have Watchtower needlessly ping Docker hub or I can leave it once per day but then I have to wait long to have production server updated. None of these options are optimal. 

Can I send a signal from Github Actions pipeline to Watchtower to trigger update or run  Watchtower in Github Actions pipeline itself?

Do you have some useful similar example?

**Comment by nemanjam** [comment, marked as answer]
It can be done with this ssh actions and `--run-once` option, what do you think?

https://github.com/marketplace/actions/ssh-remote-commands

**Reply by piksel** [reply]
You can also use the HTTP API, but it would still require an SSH connection or exposing watchtower to the internet (which is **not** recommended).
